betty/wiki/client.py

```diff
@@ -186,9 +197,24 @@ async def get_image(self, page_language: str, page_name: str) -> Image | None:
                         f"Could not successfully parse the JSON content returned by {url}: {error}"
                     )
                 ) from error
+
+            # Fetch copyright information
+            copyright = None
+            try:
+                artist = image_info["extmetadata"]["Artist"]["value"]
+                # Strip any HTML tags from the artist field, as it may contain markup
+                clean_artist = re.sub(r"<[^>]+>", "", unescape(artist)).strip()
```

Comment:
Can we use html5lib for that? It's already a Betty dependency, and HTML is complex enough we don't want to do this ourselves and get it wrong.